torch/nn/parallel/distributed.py

```diff
@@ -639,6 +639,10 @@ def _register_comm_hook(self, state: object, hook: callable):
                              torch._C.Future associated with the completion of
                              c10d.ProcessGroupNCCL.work.
 
+        .. warning ::
+            GradBucket bucket's tensors will not be predivided by world_size. User is
```

Comment:
nit: mentioning bucket twice here, probably say GradBucket's tensors will ....

Reply:
just replaced with grad bucket to be consistent with other parts of the API.